addons/better-img-uploads/userscript.js

```diff
@@ -25,7 +25,7 @@ export default async function ({ addon, console, msg }) {
     });
     button.append(img);
     const input = Object.assign(document.createElement("input"), {
-      accept: ".svg, .png, .bmp, .jpg, .jpeg",
+      accept: ".svg, .png, .bmp, .jpg, .jpeg, .sprite2, .sprite3",
```

Comment:
Note that this didn't fix a bug per-se, but it tells the OS file picker to accept sprite2 and sprite3 file uploads.